<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPlot</class>
 <widget class="QMainWindow" name="MainPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainPlot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:1 rgba(0, 59, 50, 255), stop:0 rgba(0, 69, 60, 255))
}

</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCustomPlot" name="widget_plot" native="true">
      <property name="styleSheet">
       <string notr="true">border-radius: 5px</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_control">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_start">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(0, 129, 120);
	color: rgb(195,204,221);
	border-radius: 3px;
}

QPushButton::hover {
	background-color: rgb(0,149,140);
    border: 1px solid rgb(1, 177, 230)
}

QPushButton::pressed {
	background-color: rgb(0, 169, 160);
}

QPushButton::disabled {
	background-color: rgb(0, 89, 80);
}</string>
         </property>
         <property name="text">
          <string>Старт</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/start.webp</normaloff>:/icons/start.webp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pause">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(0, 129, 120);
	color: rgb(195,204,221);
	border-radius: 3px;
}

QPushButton::hover {
	background-color: rgb(0,149,140);
    border: 1px solid rgb(1, 177, 230);
}

QPushButton::pressed {
	background-color: rgb(0, 169, 160);
}

QPushButton::disabled {
	background-color: rgb(0, 89, 80);
}</string>
         </property>
         <property name="text">
          <string>Пауза</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/pause.webp</normaloff>:/icons/pause.webp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_stop">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(0, 129, 120);
	color: rgb(195,204,221);
	border-radius: 3px;
}

QPushButton::hover {
	background-color: rgb(0,149,140);
    border: 1px solid rgb(1, 177, 230);
}

QPushButton::pressed {
	background-color: rgb(0, 169, 160);
}

QPushButton::disabled {
	background-color: rgb(0, 89, 80);
}</string>
         </property>
         <property name="text">
          <string>Стоп</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
